test/services/user/services/AdminUsers.test.js

```diff
@@ -1002,6 +1002,38 @@ describe('AdminUsersService', () => {
 		it('do not update account if from external system (teacher, patch)', () =>
 			doNotUpdateAccountIfSystemIdIsSet('teacher', 'patch', adminTeachersService));
 
+		const doNotPatchUserIfExternallyManaged = (role, type, service) => async () => {
+			const school = await testObjects.createTestSchool({
+				name: 'testSchool1',
+				source: 'notInternal',
+			});
+			const user = await testObjects.createTestUser({ roles: [role], schoolId: school._id });
+			const admin = await testObjects.createTestUser({ roles: ['administrator'], schoolId: school._id });
```

Comment:
admin and user have the same school id, so admin should be able to edit the student